### PR TITLE
Add Glimmer Apollo writeup

### PR DESCRIPTION
**Pull request by amyrlam**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

### CODE REVIEW

File: content/the-ember-times-issue-186.md [hunk 1]
```diff
@@ -52,15 +35,19 @@ Read more about the different strategies to avoiding lifecycle in components at
...
+Optimising image size and formats to improve performance can get complicated, but the [`ember-responsive-image`](https://github.com/kaliber5/ember-responsive-image) addon has your back! Now you have even more reason to take advantage of its abilities following the annoucement by [Simon Ihmig (@simonihmig)](https://github.com/simonihmig) of [version 3](https://github.com/kaliber5/ember-responsive-image/releases/tag/v3.0.0). This release provides support for Cloudinary and imgix image CDNs, further enriching a full set of functionality for re-sizing images and optimizing image loading times across all browsers. Go check it out.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+---
+
+## [Announcing Glimmer Apollo 🎉](https://glimmer-apollo.com/)
+
+[Jose Marluedke (@josemarluedke)](https://github.com/josemarluedke) shared [Glimmer Apollo](https://glimmer-apollo.com/) on [Twitter](https://twitter.com/josemarluedke/status/1410295473072611328), a new addon that integrates GraphQL in your Ember and Glimmer apps with a declarative API to query, mutate, and access GraphQL data.
```
Comment (amyrlam) [inline content/the-ember-times-issue-186.md]: @josemarluedke if you have anything you want added / edited, please let me know! hoping to get the issue out tomorrow in the PM pacific time (got delayed due to being under the weather). it sounds really cool just not sure if i 100% follow everything yet

btw, the docs site looks great, it's docfy?

Comment (josemarluedke) [reply content/the-ember-times-issue-186.md]: Just a correction to my name. It's Josemar Luedke.

```suggestion
[Josemar Luedke (@josemarluedke)](https://github.com/josemarluedke) shared [Glimmer Apollo](https://glimmer-apollo.com/) on [Twitter](https://twitter.com/josemarluedke/status/1410295473072611328), a new addon that integrates GraphQL in your Ember and Glimmer apps with a declarative API to query, mutate, and access GraphQL data.
```

File: content/the-ember-times-issue-186.md [hunk 2]
```diff
@@ -52,15 +35,19 @@ Read more about the different strategies to avoiding lifecycle in components at
...
+
+For background, [GraphQL](https://graphql.org/) is a query language for APIs and a runtime for fulfilling those queries with your existing data. [Apollo Client](https://www.apollographql.com/docs/react/) is a popular declarative data fetching library for GraphQL. Glimmer Apollo uses the concept of [Resources](https://www.pzuraq.com/introducing-use/) to enable the integration of Glimmer's autotracking system with Apollo Client. The library is also reactive: queries are automatically re-executed when arguments change. The UI reflects any changes when cache data is updated.
+
+Glimmer Apollo is written in TypeScript, allowing APIs to specify the shape of the result data and variables. TypeScript also has friendly IntelliSense while coding. Glimmer Apollo is also an **Embroider native addon** (v2) and uses FastBoot (server-side rendering). 💯
```
Comment (josemarluedke) [inline content/the-ember-times-issue-186.md]:
```suggestion
Glimmer Apollo is written in TypeScript, allowing APIs to specify the shape of the result data and variables. TypeScript also has friendly IntelliSense while coding. Glimmer Apollo is also an **Embroider native addon** (v2) and integrates with FastBoot (server-side rendering). 💯
```

File: content/the-ember-times-issue-186.md (hunk 1 above)
Comment (amyrlam) [reply content/the-ember-times-issue-186.md]: sorry, bad copy pasta on my part! 🤦‍♀️

Comment (josemarluedke) [reply content/the-ember-times-issue-186.md]: > sorry, bad copy pasta on my part! 🤦‍♀️

No worries.

> btw, the docs site looks great, it's docfy?

Thanks. Yes, it's Docfy. The UI is built on a theme (ember addon) that is intended to be shared and usable with other docfy apps, I just need to finish that up. https://github.com/josemarluedke/docfy-theme-ember

File: content/the-ember-times-issue-186.md [hunk 3]
```diff
@@ -53,15 +48,19 @@ Read more about the different strategies to avoiding lifecycle in components at
...
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+---
+
+## [Announcing Glimmer Apollo 🎉](https://glimmer-apollo.com/)
+
+[Josemar Luedke (@josemarluedke)](https://github.com/josemarluedke) shared [Glimmer Apollo](https://glimmer-apollo.com/) on [Twitter](https://twitter.com/josemarluedke/status/1410295473072611328), a new addon that integrates GraphQL in your Ember and Glimmer apps with a declarative API to query, mutate, and access GraphQL data.
+
+For background, [GraphQL](https://graphql.org/) is a query language for APIs and a runtime for fulfilling those queries with your existing data. [Apollo Client](https://www.apollographql.com/docs/react/) is a popular declarative data fetching library for GraphQL. Glimmer Apollo uses the concept of [Resources](https://www.pzuraq.com/introducing-use/) to enable the integration of Glimmer's autotracking system with Apollo Client. The library is also reactive: queries are automatically re-executed when arguments change. The UI reflects any changes when cache data is updated.
```
Comment (ijlee2) [inline content/the-ember-times-issue-186.md]:
```suggestion
For background, [GraphQL](https://graphql.org/) is a query language for APIs and a runtime for fulfilling those queries with your existing data. [Apollo Client](https://www.apollographql.com/docs/react/) is a popular, declarative data-fetching library for GraphQL. Glimmer Apollo uses the concept of [Resources](https://www.pzuraq.com/introducing-use/) to enable the integration of Glimmer's autotracking system with Apollo Client. The library is also reactive: queries are automatically re-executed when arguments change. The UI reflects any changes when cache data are updated.
```

**Review by ijlee2 — APPROVED** [review]
Looks great, thank you both for helping with sharing the news!

File: content/the-ember-times-issue-186.md (hunk 3 above)
Comment (amyrlam) [reply content/the-ember-times-issue-186.md]: i think is is grammatically correct, but can change it to "when cache data has been updated" upstream

**Pull request merged**